openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1050,18 +1061,29 @@ The Batch Credential Endpoint issues multiple Credentials in one Batch Credentia
 
 Communication with the Batch Credential Endpoint MUST utilize TLS. 
 
-The Client can request issuance of multiple Credentials of certain types and formats in one Batch Credential Request. This includes Credentials of the same type and multiple formats, different types and one format, or both. 
+The Client can request issuance of multiple Credentials that can be:
 
-## Batch Credential Request {#batch-credential_request}
+* different Credential instances of the different Credential Configurations
+* different Credential instances of the same Credential Configuration but the different Credential Dataset
+* different Credential instances of the same Credential Configuration and the same Credential Dataset (with different cryptographic material)
+* a combination of the points above
 
-The Batch Credential Endpoint allows a Client to send multiple Credential Request objects (see (#credential-request)) to request the issuance of multiple Credentials at once. A Batch Credential Request MUST be sent as a JSON object using the `application/json` media type.
+Wallets need to differentiate between these options because they can lead to variations in user experience. While Credentials with the same Credential Configuration but different Credential Dataset should be displayed separately (e.g. vehicle registration credentials for multiple cars), Credentials with the same Credential Configuration and Credential Dataset (e.g. multiple age proofs for unlinkability) should only appear as a single Credential in the Wallet user interface, as these technical details are not relevant to the average End-user. 
 
-The following parameters are used in the Batch Credential Request:
+## Batch Credential Request {#batch-credential-request}
 
-* `credential_requests`: REQUIRED. Array that contains Credential Request objects, as defined in (#credential-request). The individual Credential Request objects MUST NOT contain `credential_response_encryption`.
+A Client submits a Batch Credential Request to the Batch Credential Endpoint by sending a JSON object containing the following parameters in the entity-body of an HTTP POST request, using the `application/json` media type.
+
+* `credential_requests`: REQUIRED. Array that contains Batch Credential Request objects, each of these objects may refer to a different Credential Configuration or the same Credential Configuration with a different Credential Dataset. A Batch Credential Request object contains either a `format` parameter (and possibly further Credential Format specific parameters) or a `credential_identifier` parameter, an optional `proofs` parameter, but no `proof` or `credential_response_encryption` parameter.
+  * `format`: REQUIRED if the `credential_identifiers` parameter was not returned from the Token Response. It MUST NOT be used otherwise. See (#credential-request).
```

Comment:
The three parameters here are essentially re-used from above. I think that a credential request data structure should be explicitly defined above for the single-credential endpoint and then that definition can be re-used here.